conans/client/loader_txt.py

```diff
@@ -52,7 +58,7 @@ def _parse_args(param_string):
             return root_package, ignore_case, folder, excludes, keep_path
 
         def _parse_import(line):
-            pair = line.split("->")
+            pair = line.split("->", 1)
```

Comment:
may be add check here like above (what if there is no "->" within string)?

Reply:
also, what if "->" is part of the path?

Reply:
That makes an invalid path, not possible.

Reply:
> may be add check here like above (what if there is no "->" within string)?

Done, added new exception and error message, and test.